### ISSUE TITLE
Suggested found patient in an autocomplete dropdown

### ISSUE THREAD

**Issue by clemens-msupply** [openmsupply/open-msupply#2296]
In https://github.com/openmsupply/open-msupply/issues/1840 the patient matches are shown in a separate dropdown menu.

- [ ] Discuss final design
- [ ] Depending on previous point, implement an autocomplete dropdown for the current input field

**Comment by richardmoizeau** [comment]
a few additional comments here: 

- [ ] I thought user would first search for the mother's record in a list of patients and then data would be automatically populated. 
- [ ] have the same lookup functionality in the Early Infant Diagnosis program
![Screenshot 2023-10-03 at 10 14 04 AM](https://github.com/openmsupply/open-msupply/assets/74992958/0d4dacd5-4147-4fb0-9d35-f98ad636787e)

**Comment by clemens-msupply** [comment]
@richardmoizeau yes this is still part of https://github.com/openmsupply/open-msupply/issues/2189

**Comment by DhanyaHerath** [comment]
Closing as part of a previous project
